lib/awaitableSender.ts

```diff
@@ -181,22 +178,18 @@ export class AwaitableSender extends BaseSender {
    * @param {Message | Buffer} msg The message to be sent. For default AMQP format msg parameter
    * should be of type Message interface. For a custom format, the msg parameter should be a Buffer
    * and a valid value should be passed to the `format` argument.
-   * @param {Buffer | string} [tag] The message tag if any.
-   * @param {number} [format] The message format. Specify this if a message with custom format needs
-   * to be sent. `0` implies the standard AMQP 1.0 defined format. If no value is provided, then the
-   * given message is assumed to be of type Message interface and encoded appropriately.
    * @param {AwaitableSendOptions} [options] Options to configure the timeout and cancellation for
```

Comment:
Can we indicate that AwaitableSendOptions is now much more than timeout and cancellation?

Reply:
Updated. Also made the same change for the regular Sender.